lib/dfp_helper.rb

```diff
@@ -15,11 +15,12 @@ def dfp_helper_slot(_i, options = {})
       _id = options[:div_id]
       _id ||= "div-gpt-ad-#{@@dfp_helper_id}-#{dfp_helper_slots.size}"
       _size = options[:size] || _i.match(/\d+x\d+/)[0].split('x')
+      _styles = options[:responsive_mapping] ? '' : "width:#{_size[0]}px; height:#{_size[1]}px;"
```

Comment:
yeah i know. I think that the underscored variables are the least of this codes problems. I could refactor but maybe backlog that since ya know, eveyrthing else that needs to get done today.